conans/client/build/autotools_environment.py

```diff
@@ -339,6 +341,13 @@ def append(*args):
         tmp_compilation_flags = copy.copy(self.flags)
         if self.fpic:
             tmp_compilation_flags.append(pic_flag(self._conanfile.settings))
+        if tools.is_apple_os(self._os):
+            concat = " ".join(tmp_compilation_flags)
+            if os.environ.get("CFLAGS", None):
```

Comment:
Couldn't the ``version-min`` come defined also in CXXFLAGS? I think it would make sense check that var too, just in case, as C++ users are probably defining it in that var, what do you think?

Reply:
Done